src/sdam/topology_description.ts

```diff
@@ -64,22 +64,10 @@ export class TopologyDescription {
     this.compatible = true;
     this.heartbeatFrequencyMS = options.heartbeatFrequencyMS ?? 0;
     this.localThresholdMS = options.localThresholdMS ?? 15;
-
-    if (setName) {
-      this.setName = setName;
-    }
-
-    if (maxSetVersion) {
-      this.maxSetVersion = maxSetVersion;
-    }
-
-    if (maxElectionId) {
-      this.maxElectionId = maxElectionId;
-    }
-
-    if (commonWireVersion) {
-      this.commonWireVersion = commonWireVersion;
-    }
+    this.setName = setName ?? null;
+    this.maxElectionId = maxElectionId ?? null;
+    this.maxSetVersion = maxSetVersion ?? null;
```

Comment:
Defaulted in the constructor so could be simplified.

```suggestion
    this.setName = setName;
    this.maxElectionId = maxElectionId;
    this.maxSetVersion = maxSetVersion;
```